### DISCUSSION TITLE
Styling of the default popover container

### DISCUSSION THREAD

**Discussion by yaforster** [ng-matero/extensions#336] [Q&A] [answered]
Hi there!

I am building a site using angular material and just recently came across the extensions and I would like to use popover as way to show a picture when I mouseover a div.

I have attached a popover that contains an image. I have not knowingly styled the popover in any way, but when I mouse over my element and the popover is displayed there is a margin around my own content, a transparent box, and a shadow.

![image](https://github.com/user-attachments/assets/7edc7824-16a4-4b2a-b8a0-e543d4a56993)

What styles do I need to overwrite to remove this box?

Also note that I just updated from 17.3.5 to 18.2.0 where the behaviour of this box seemingly changed. On 17.3.5, I can only see it after the popover has been displayed once. On 18.2.0 it is always visible.

**Comment by nzbin** [comment, marked as answer]
Do you define the theme correctly?
https://github.com/ng-matero/extensions?tab=readme-ov-file#theming

**Comment by yaforster** [comment]
I did not. Since I was intending on styling the popover myself, I naively thought I could skip that step...
It works now! Thanks a lot!
